susePublicCloudInfoClient/lib/susepubliccloudinfoclient/infoserverrequests.py

```diff
@@ -60,29 +142,106 @@ def __get_base_url():
 def __get_data(url):
     """Make the request and return the data or None in case of failure"""
     response = requests.get(url)
+    print url
```

Comment:
I don't think we  want to print the url by default, maybe if there is a --verbose switch on the command line. Printing it by default will mess up automated processing of the result as it adds a line to the output that is not data.